src/App.tsx

```diff
@@ -1,17 +1,21 @@
-import { SignInButton, SignedIn, SignedOut, UserButton } from "@clerk/clerk-react"
+import {
+  SignInButton,
+  SignedIn,
+  SignedOut,
+  UserButton,
+} from '@clerk/clerk-react';
 
 function App() {
-
   return (
     <div>
       <SignedOut>
         <SignInButton />
       </SignedOut>
       <SignedIn>
-        <UserButton afterSignOutUrl="/" />
+        <UserButton />
       </SignedIn>
     </div>
```

Comment:
```suggestion
    </header>
```
Change to match the Quickstart guide